mitodl/hugo-course-publisher
# Course zip static asset support

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/hugo-course-publisher/issues/249

#### What's this PR do?
This PR alters the `build_course_zips` build script so that its arguments come from environment variables, and adds support for gathering and bundling static assets (PDF's, etc.) via the `COURSE_ZIPS_STATIC_ASSET_PATH` and `COURSE_ZIPS_STATIC_PREFIX` env variables.

#### How should this be manually tested?
Following the updated instructions in the readme, you should be able to set the environment variables necessary for building the course zips with static assets, and the resulting zips should contain all the static assets for each course.  Note that when browsing the course site from the zip from the local file system, the inline PDF viewer will not work due to a CORS issue.  The download button should still work, and the static files should all be located in the zip at the root level by default, or in the path specified by `COURSE_ZIPS_STATIC_PREFIX`.
